corehq/form_processor/models.py

```diff
@@ -1218,14 +1218,14 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges):
     objects = RestrictedManager()
 
     domain = models.CharField(max_length=255, null=False, default=None)
-    case_id = models.CharField(max_length=255, db_index=True, default=None)  # remove foreign key until we're sharding this
+    case_id = models.CharField(max_length=255, default=None)  # remove foreign key until we're sharding this
```

Comment:
This is sharded by `case_id` so we could make it a foreign key which I guess we should. I'll update.